spec/Section 4 -- Composition.md

```diff
@@ -18,4 +18,66 @@ run in sequence to produce the composite execution schema.
 
 ### Post Merge Validation
 
+#### Empty Merged Object Type
+
+**Error Code**
+
+`Empty_Merged_Object_Type`
+
+**Formal Specification**
+
+- Let {types} be the set of all object types across all source schemas
+- For each {type} in {types}:
+  - {IsObjectTypeEmpty(type)} must be false.
+
+IsObjectTypeEmpty(type):
+
+- Let {fields} be a set of all fields of all types with coordinate and kind
+  {type} across all source schemas
```

Comment:
I'm not following the "with coordinate and kind {type}" part.

Reply:
done